spec/spec_helper.rb

```diff
@@ -13,8 +13,15 @@ def find_by_title(docs, title)
     docs.find { |d| d.data["title"] == title }
   end
 
-  def index_fixture
-    File.read File.expand_path("fixtures/index.html", __dir__)
+  %w(
+    index
+    minified
+    plain_index
+    plain_minified
+  ).each do |method_name|
+    define_method "#{method_name}_fixture".to_sym do
```

Comment:
In Jekyll we do `fixture(“plain_index.html”)`. I don’t love the metaprogramming here. Could be use a method argument instead?